test/integration_test/lib/analysis/author/test_ranking.py

```diff
@@ -0,0 +1,19 @@
+from lib.analysis.author.ranking import *
+
+
+def test_get():
+
+    json_filename = './test/integration_test/data/headers.json'
+    output_filename = './test/integration_test/data/author_ranking.csv'
+    req_ranking = [('opensuse-kernel@opensuse.org', 5), ('aiaredir@yahoo.com', 4), ('jeffm@suse.com', 3), ('aj@suse.de', 2)]
+    req_csv = """Email Address,Author's Score,Author's Rank
+opensuse-kernel@opensuse.org,5,1
```

Comment:
The usual convention for indentation is to offset the following lines to align with the string that you are concatenating. In this particular case, the vertical alignment can happen at Email word.